code/consistent_sampler.py

```diff
@@ -373,35 +373,42 @@ def next_ticket(ticket):
                   ticket.generation+1)
```

Comment:
one thing I'm not sure about is whether i need to copy this file to `pkg` as well in order to import it